p/themes/Nord/nord.css

```diff
@@ -754,7 +750,6 @@ li.item.active {
 /*===========*/
 .category .title.error::before {
 	color: var(--accent-light);
-	content: "⚠ ";
```

Comment:
This stuff is already set in frss.css I assume? :-)

Reply:
yes, it is:

https://github.com/FreshRSS/FreshRSS/blob/af8480651dea478e2a60dc13b9ea44d364d0f7b7/p/themes/base-theme/frss.css#L1810-L1815